### ISSUE TITLE
Allow filtering for tags

### ISSUE THREAD

**Issue by wesbos** [akashnimare/ReadAMA#1]
on my AMA I tag them with 'good questions' so we can wade through and get the good ones. Would be cool to add this!

Great app :)

**Comment by akashnimare** [comment]
Yes, I have this feature in my todo list. Will update it. Thanks :)

**Comment by akashnimare** [comment]
@wesbos just little update. As of now I have added showing labels on issue.
![labels](https://cloud.githubusercontent.com/assets/2263909/16903209/320df790-4c95-11e6-95b1-82d68ed45eb3.png)
